airbyte-cdk/python/unit_tests/sources/file_based/stream/concurrent/test_adapters.py

```diff
@@ -201,19 +208,6 @@ def test_json_schema_is_delegated_to_wrapped_stream(self):
         assert self._facade.get_json_schema() == json_schema
         self._abstract_stream.get_json_schema.assert_called_once_with()
 
-    def test_given_cursor_is_noop_when_supports_incremental_then_return_legacy_stream_response(self):
-        assert (
-            FileBasedStreamFacade(
-                self._abstract_stream, self._legacy_stream, _ANY_CURSOR, Mock(spec=SliceLogger), Mock(spec=logging.Logger)
-            ).supports_incremental
-            == self._legacy_stream.supports_incremental
-        )
-
-    def test_given_cursor_is_not_noop_when_supports_incremental_then_return_true(self):
-        assert FileBasedStreamFacade(
-            self._abstract_stream, self._legacy_stream, Mock(spec=Cursor), Mock(spec=SliceLogger), Mock(spec=logging.Logger)
-        ).supports_incremental
-
```

Comment:
These pass, but didn't make sense to me anymore - any pointers would be appreciated